modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_users_controller.rb

```diff
@@ -4,7 +4,39 @@ module AccreditedRepresentativePortal
   module V0
     class RepresentativeUsersController < ApplicationController
       def show
-        render json: @current_user
+        # TODO: Move to serializer. Temporary quick and dirty.
+        render json: {
+          account: {
+            # Correct value?
+            account_uuid: @current_user.uuid,
+          },
+          profile: {
+            first_name: @current_user.first_name,
+            last_name: @current_user.last_name,
+            verified:,
+          },
+          # TODO: Should they get prefill for e.g. 21a?
```

Comment:
IMO yes, we should prefill what's possible (likely some personal info, phone, email address. maybe a home address on file)

Reply:
It seems like `prefills_available` is actually a list of form IDs (so this should just be `["21a"]`)

I'd say we keep this as is and implement prefill in another PR

Reply:
We have a few options for getting a new rep's contact info to consider: OAuth, MPI, or VA Profile. If the rep is also a Veteran or claimant, MPI and VA Profile would give us more complete info than OAuth, but OAuth is probably an easier lift.

Reply:
Either remove TODO comment or call out that one can see how we're foregoing prefill for now with how this array is empty. But for someone familiar with the system, that comment might be extraneous.